apps/fabric-website/src/pages/Styles/Colors/docs/web/ColorsNeutrals.md

```diff
@@ -1 +1,5 @@
 These are commonly used for backgrounds, strokes, and interactive states within controls.
+
+This palette can be used to customize your `theme.palette`. See how our default `theme.palette` maps to new Fluent colors with this [web color conversion table](https://static2.sharepointonline.com/files/fabric/fabric-website/files/fabric-neutrals-web-color-conversion.pdf).​​
+
+In case you have a need to create a custom neutral theme palette to use in your application, you can use the [Theme Designer](aka.ms/themedesigner).​
```

Comment:
Maybe not needed but just in case:
```suggestion
In case you have a need to create a custom neutral theme palette to use in your application, you can use the [Theme Designer](https://aka.ms/themedesigner).​
```